src/css/components/_c-copy-email-button.scss

```diff
@@ -9,7 +9,7 @@
 	font-size: 0.75rem;
 	letter-spacing: $font-tracking-slight;
 	margin-left: 1ch;
-	padding: 0.2rem 0.4rem 0.1rem 0.4rem;
```

Comment:
And here!

Reply:
Same as [prior response](https://github.com/a11yproject/a11yproject.com/pull/984/files/a442075085a0173abbc13b41848ff2affdf3df4c#r464021536).

Please advise on direction for all and I will adjust accordingly.